tests/fsharpqa/Source/run.pl

```diff
@@ -1,4 +1,4 @@
-# run.pl
```

Comment:
Perl scripts don't run on non-windows machines, so while change is correct, it makes a very little sense